docs/modules/ROOT/pages/how-tos/logging/increase-elasticsearch-storage-size.adoc

```diff
@@ -0,0 +1,283 @@
+= Increase Storage Size of an OCP Logging Elasticsearch instance
+
+[abstract]
+This page describes how to increase the underlying storage size of the OpenShift Cluster Logging Elasticsearch instance.
+
```

Comment:
I think we should mention the impact of the resize.
```suggestion

[IMPORTANT]
This is a disruptive operation!
During the resize, the Elasticsearch cluster will experience downtime for a few minutes to hours.

```

Reply:
There is no downtime since we have single redundancy!

Reply:
There will be reduced performance for a few minutes though and new logs are delayed. I try to write it like this.